app/src/main/res/values/strings.xml

```diff
@@ -762,8 +762,8 @@
     <string name="dns_btm_record_type">Query type: %1$s</string>
 
     <!-- Home Screen activity -->
-    <string name="hs_download_positive_default">ok</string>
-    <string name="hs_download_positive_play_store">Complete Update</string>
+    <string name="hs_download_positive_default">OK</string>
+    <string name="hs_download_positive_play_store">Complete installation</string>
```

Comment:
it isn't an install, but an update (keeping it consistent with how Google Play uses those terms).

Reply:
In my head that is an upgrade, and I try to avoid that because it isn't a common distinction to make.
Is anything else possible?